clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/WgsCramTests.scala

```diff
@@ -12,23 +12,19 @@ import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
   ElasticsearchUtil
 }
 import org.broadinstitute.clio.transfer.model.wgscram._
-import org.broadinstitute.clio.util.model.{
-  DocumentStatus,
-  Location,
-  RegulatoryDesignation,
-  UpsertId
-}
+import org.broadinstitute.clio.util.model._
 import org.scalatest.Assertion
 
 import scala.concurrent.Future
 
+//TODO This should be deleted once we're using the new Cram API in all places
```

Comment:
Can we drop most of the tests in here? The cram tests take a particularly long time and duplicating everything seems unnecessary since most of the logic is shared. I think we're covered on supporting old routes/commands as long as we have a bare bones test for each command.

Reply:
We don't run the integration tests all that often. I just kept the old `wgs-cram` tests and copied them over and changed everything to the `cram` API. It seems like a lot more work to thread things through to use both the old and new commands in the same tests. This way, once we're done with the old API, we can just delete the whole file. No need to untangle a bunch of code.